gems/sorbet/lib/constant_cache.rb

```diff
@@ -138,7 +138,7 @@ def name_by_class(klass)
         rescue LoadError => err
```

Comment:
Would it make more sense for this rescue to go up here? I don't have enough understanding/context to know which is more appropriate.